Publishmodel [DDO-770] [DDO-903]

emandakh opened the pull request:
Preliminary public site datamodel.terra.bio with content negotiation based on Apache server container.

kreinold reviewed (changes requested):
We don't need the html files for the subsets. 

I can't really comment on the yaml, Docker or gcloud but I have tested them and they all work as expected.

emandakh commented:
- removed the HTML files for the subsets 
- added 200 ProxyPass response instead of 303 Redirect response
- added handling of "Accept: text/turtle, application/rdf+xml" and default to text/turtle

Tested http://datamodel.terra.bio/UBERON_subset and it's working
